File: src/BubbleSort/Python/BubbleSort.py
def bubble_sort(arr):
    n = len(arr)

    for i in range(n):

        for j in range(0, n - i - 1):
            if arr[j] > arr[j + 1]:
                arr[j], arr[j + 1] = arr[j + 1], arr[j]


arr = [12, 43, 27, -10, 49, 15, 15]

bubble_sort(arr)

print("Bubble Sort array is:")
for i in range(len(arr)):
    print("%d" % arr[i])

File: src/Sort/BubbleSort/Java/BubbleSort.java
package Sort.BubbleSort.Java;

import java.util.Arrays;

public class BubbleSort {

  private static int arr[] = {12, 54, 98, 45, 32, 85, 9, 122, 2};


  public static void main(String[] args) {
    bubbleSort(arr);
    System.out.println("Sorted array is: ");
    printArray(arr);
  }

  private static void bubbleSort(int[] arr) {

    for (int i = 0; i < arr.length - 1; i++) {
      for (int j = 0; j < arr.length - 1 - i; j++) {
        if(arr[j] > arr[j + 1]) {
          int tmp = arr[j];
          arr[j] = arr[j + 1];
          arr[j + 1] = tmp;
        }
      }

    }
  }

  private static void printArray(int[] arr) {
    System.out.println(Arrays.toString(arr));
  }

}
